conf/setup.sql

```diff
@@ -1,5 +1,5 @@
 # Create the DB
-CREATE DATABASE css;
+CREATE DATABASE css character set UTF8mb4 collate utf8mb4_bin;
```

Comment:
I ran what you listed above and got the following.

+--------------------------+-------------------+
| Variable_name            | Value             |
+--------------------------+-------------------+
| character_set_client     | utf8              |
| character_set_connection | utf8              |
| character_set_database   | latin1            |
| character_set_filesystem | binary            |
| character_set_results    | utf8              |
| character_set_server     | latin1            |
| character_set_system     | utf8              |
| collation_connection     | utf8_general_ci   |
| collation_database       | latin1_swedish_ci |
| collation_server         | latin1_swedish_ci |
+--------------------------+-------------------+